package.json

```diff
@@ -42,14 +39,20 @@
     "curlable": "./curlable.js"
   },
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha"
+    "test": "nyc --reporter=html --reporter=text --all mocha"
```

Comment:
Two reporters intentional?

Reply:
Yes, text for console (short summary) and html for browser (interactive). At work I use three, two into console (summary and per-file summary) but still use the browser one to see uncovered code lines.